packages/core/src/utilities/getPixelSpacingInformation.ts

```diff
@@ -41,9 +43,14 @@ function calculateRadiographicPixelSpacing(instance) {
 
   if (!PixelSpacing) {
     if (!EstimatedRadiographicMagnificationFactor) {
-      console.warn(
-        'EstimatedRadiographicMagnificationFactor was not present. Unable to correct ImagerPixelSpacing.'
-      );
+      if (!alreadySeenWarn.has(instance.SeriesInstanceUID)) {
+        console.warn(
```

Comment:
Not part of this change really, just that I wasn't able to debug one particular study issue because of a wall of messages.